src/components/modals/MovieForm.js

```diff
@@ -1,7 +1,146 @@
+                        </div>
+
+                        <div className="row-container">
+                            <label>
+                                <p className="input-label">MOVIE URL*</p>
+                                <input
+                                    className="input-form-movie"
+                                    name="poster_path"
+                                    type="text"
+                                    placeholder="Movie URL here"
+                                    value={movieObj.poster_path}
+                                />
+                            </label>
+                        </div>
+
+                        <div className="row-container">
+                            <label>
+                                <p className="input-label">GENRES*</p>
+                                <select
+                                    className="input-form-select"
+                                    name="genres"
+                                    placeholder="Select Genre"
+                                    value={movieObj.genres}>
+                                    <option value="">Select Genre</option>
+                                    <option value="Crime">Crime</option>
+                                    <option value="Documentary">Documentary</option>
+                                    <option value="Horror">Horror</option>
+                                    <option value="Comedy">Comedy</option>
+                                </select>
+                            </label>
+                        </div>
+
+                        <div className="row-container">
+                            <label>
+                                <p className="input-label">RUNTIME*</p>
+                                <input
+                                    className="input-form-movie"
+                                    name="runtime"
+                                    type="text"
+                                    placeholder="Runtime here"
+                                    value={movieObj.runtime}
+                                />
+                            </label>
+                        </div>
+
+                        <div className="row-container">
+                            <label>
+                                <p className="input-label">OVERVIEW*</p>
+                                <textarea
+                                    className="input-form-textarea"
+                                    name="overview"
+                                    placeholder="Overview here"
+                                    rows="20">
+                                    {movieObj.overview}
+                                </textarea>
+                            </label>
+                        </div>
+
+                        <div className="button-container">
+                            <button className="btn-reset" type="button">
+                RESET
```

Comment:
I just ran `npm run lint` and noticed that you have issues in your code.
You should not have been able to push this code if you configured Husky correctly.
Therefore:
1. Check the configs of Hustky - it should prevent you from pushing code if `npm run lint` fails
2. Fix linting issues
<img width="1171" alt="Screenshot 2022-05-09 at 21 56 07" src="https://user-images.githubusercontent.com/25229823/167497049-b75d293d-cf48-4257-a102-c1b0d1f5e8d2.png">